picamctl/picamctl.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+# akseidel 02/06/2021
+# A script automating the steps to run showmewencam's "camera-ctl".
+# The script will pause for continue or exit if run with an argument.
+
+# edit this next line to be the correct text pattern for your computer.
+portnamepat="tty.usbmodem" #typical for showmewebcam on Appl OS X
+#portnamepat="ttyACM" #probably typical for showmewebcam on Linux
+#portnamepat="tty."
+# The client is an application you want this script to also run. 
+runtheclient="true"
```

Comment:
Better as a parameter

Reply:
In the intended situation running the client is almost 100% wanted and in MacOS it would be difficult to have multiple instances. This variable is almost an ort from the early development process. An issue with parameters is that they result in what is viewed as unnecessary complication by the intended user.